### DISCUSSION TITLE
How to get cursor in recording?

### DISCUSSION THREAD

**Discussion by Ricky610329** [obsproject/obs-studio#10556] [General]
Hi~
I'm building a application in python, and I found that screenshot doesn't contain cursor. Interestingly, OBS does (in recording). did you redraw the cursor or there is another api for recording in windows?
Thanks

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!

**Reply by Ricky610329** [reply]
Got it!
Thank you

**Comment by matoi974** [comment]
Yes, desktop captures do not contain your mouse pointer and obs-studio has to draw one where it thinks it should be. Also, animated mouse pointers won't animate.

**Reply by Ricky610329** [reply]
Thanks!
